### DISCUSSION TITLE
lock file do yarn vs npm

### DISCUSSION THREAD

**Discussion by felquis** [frontendbr/forum#630] [Discussão]
Olá :)

Sabem se o yarn e npm planejam fazer um lock file portável?

yarn 0.27.5
npm 5.3.0

npm-lock.json
```txt
{
  "requires": true,
  "lockfileVersion": 1,
  "dependencies": {
    "ramda": {
      "version": "0.24.1",
      "resolved": "https://registry.npmjs.org/ramda/-/ramda-0.24.1.tgz",
      "integrity": "sha1-w7d1UZfzW43DUCIoJixMkd22uFc="
    }
  }
}
```

yarn.lock
```txt
# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
# yarn lockfile v1


ramda@^0.24.1:
  version "0.24.1"
  resolved "https://registry.yarnpkg.com/ramda/-/ramda-0.24.1.tgz#c3b7755197f35b8dc3502228262c4c91ddb6b857"
```

Vantagens de ser o mesmo formato:

* Times multidiciplinares que podem variar o uso entre yarn e npm
* ...

Bom, eles querem dizer a mesma coisa de forma diferente, como já compartilham o `package.json`, eu agradeceria se fizessem o mesmo para o lock file.

**Comment by barraponto** [comment]
Até onde eu sei, tem essa thread rolando: https://github.com/yarnpkg/yarn/issues/3614
